app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getTopics } = require("./controllers/app.controller");
+const { handleCustomErrors, handleServerErrors } = require("./errors");
+const app = express();
+app.use(express.json())
```

Comment:
It's important to only include code in a PR that meets the needs of that specific task/ticket
For that reason we shouldn't have this yet because it's only needed to access the body in a post request 🔍